pkg/machinepool/machinepool.go

```diff
@@ -1442,19 +1442,21 @@ func getMachinePoolReplicas(cmd *cobra.Command,
 				return
 			}
 		}
-	} else if interactive.Enabled() || !isReplicasSet && askForScalingParams {
+	} else {
```

Comment:
@marcolan018 I'd really like us to add a unit test for this. This is the type of regression we constantly run into, fix, but don't add a test for to prevent future regressions.

Could you add a unit test for the `getMachinePoolReplicas` function that ensures that if the user has not supplied `--replicas` flag on the CLI, that the returned `replicas` value from this function is what is currently set on the machinepool.

Reply:
@robpblake added the unit test for `getMachinePoolReplicas` func, please help to re-review

Reply:
Thanks @marcolan018